api-service/controller/controller.go

```diff
@@ -98,6 +98,8 @@ type APIControllerInterface interface {
 	GetExadataStorageErrorCountStatusStats(w http.ResponseWriter, r *http.Request)
 	// GetExadataPatchStatusStats return all statistics about the patch status of the exadata using the filters in the request
 	GetExadataPatchStatusStats(w http.ResponseWriter, r *http.Request)
+	//GetDefaultDatabasesTags return the default list of database tags from configuration
+	GetDefaultDatabasesTags(w http.ResponseWriter, r *http.Request)
```

Comment:
Are you sure that the 's' of databases in GetDefaultDatabasesTags should be present?

Reply:
I don't know. I thought about that: that tags are the default for all dbs, not only one!
If you prefer, it's the same to me :+1:

Reply:
Go for GetDefaultDatabasesTags